scripts/cat/pelts.py

```diff
@@ -898,24 +896,17 @@ def pattern_color_inheritance(self, parents: tuple = (), gender="female"):
         # ------------------------------------------------------------------------------------------------------------#
 
         # Determine pelt.
-        weights = [
-            0,
-            0,
-            0,
-            0,
-        ]  # Weights for each pelt group. It goes: (tabbies, spotted, plain, exotic)
-        for p_ in par_peltnames:
-            if p_ in Pelt.tabbies:
+        weights = [0, 0, 0, 0]  # Weights for each pelt group. It goes: (tabbies, spotted, plain, exotic)
+        for pelt_ in par_peltnames:
```

Comment:
Minor thing here and you were just imitating the original: could we get this set to `_pelt`? Our codebase rule for private members (if we use them) is underscore first iirc.

Reply:
Resolved